DESCRIPTION

```diff
@@ -15,9 +15,12 @@ LazyData: TRUE
 Imports:
   stringi,
   stringr,
-  jsonlite
+  jsonlite,
+  ggplot2,
```

Comment:
Heavy dependencies, especially when there are very few packages necessary so far. Maybe it's better to make this whole ggplot thing optional. I think we had this solution for mortAAR already, where ggplot2 is only in the suggests.

Reply:
I absolutely agree. But putting the packages under suggests results in an 

```
> checking package dependencies ... ERROR
  Namespace dependencies not required: 'ggplot2', 'ggridges', 'methods'
```

Any suggestions appreciated!

Reply:
See below.

Reply:
see below.